packages/lit/package.json

```diff
@@ -18,120 +18,198 @@
+      "types": "./development/directives/map.d.ts",
+      "development": "./development/directives/map.js",
       "default": "./directives/map.js"
     },
     "./directives/range.js": {
+      "types": "./development/directives/range.d.ts",
+      "development": "./development/directives/range.js",
       "default": "./directives/range.js"
     },
     "./directives/ref.js": {
+      "types": "./development/directives/ref.d.ts",
+      "development": "./development/directives/ref.js",
       "default": "./directives/ref.js"
     },
     "./directives/repeat.js": {
+      "types": "./development/directives/repeat.d.ts",
+      "development": "./development/directives/repeat.js",
       "default": "./directives/repeat.js"
     },
     "./directives/style-map.js": {
+      "types": "./development/directives/style-map.d.ts",
+      "development": "./development/directives/style-map.js",
       "default": "./directives/style-map.js"
     },
     "./directives/template-content.js": {
+      "types": "./development/directives/template-content.d.ts",
+      "development": "./development/directives/template-content.js",
       "default": "./directives/template-content.js"
     },
     "./directives/unsafe-html.js": {
+      "types": "./development/directives/unsafe-html.d.ts",
+      "development": "./development/directives/unsafe-html.js",
       "default": "./directives/unsafe-html.js"
     },
     "./directives/unsafe-svg.js": {
+      "types": "./development/directives/unsafe-svg.d.ts",
+      "development": "./development/directives/unsafe-svg.js",
       "default": "./directives/unsafe-svg.js"
     },
     "./directives/until.js": {
+      "types": "./development/directives/until.d.ts",
+      "development": "./development/directives/until.js",
       "default": "./directives/until.js"
     },
     "./directives/when.js": {
+      "types": "./development/directives/when.d.ts",
+      "development": "./development/directives/when.js",
       "default": "./directives/when.js"
     },
     "./experimental-hydrate-support.js": {
+      "types": "./development/experimental-hydrate-support.d.ts",
+      "development": "./development/experimental-hydrate-support.js",
       "default": "./experimental-hydrate-support.js"
     },
     "./experimental-hydrate.js": {
+      "types": "./development/experimental-hydrate.d.ts",
+      "development": "./development/experimental-hydrate.js",
       "default": "./experimental-hydrate.js"
     },
     "./html.js": {
+      "types": "./development/html.d.ts",
+      "development": "./development/html.js",
       "default": "./html.js"
     },
     "./polyfill-support.js": {
+      "types": "./development/polyfill-support.d.ts",
+      "development": "./development/polyfill-support.js",
       "default": "./polyfill-support.js"
     },
     "./static-html.js": {
+      "types": "./development/static-html.d.ts",
+      "development": "./development/static-html.js",
       "default": "./static-html.js"
```

Comment:
I don't think we need development for lit, because all the differences are I'm the underlying packages

Reply:
I agree that it's no different than the default build. The types field does need to point to the `./development/*` directory for the self reference to work during build time since that is the specified `outDir` so I did have to add the entire `/development/` directory to the published package. We could possibly just include the `d.ts` and `d.ts.map` files there.

I noticed some of the labs packages also include the (unnecessary) `"development"` export probably just from copy pasting so I opted to just follow that here too.

An alternative, and perhaps arguably more sensible, thing to do might be to change the tsc outDir to emit directly to the package root and not do the treemirror copy for the packages that don't need a development build but that seemed like extra work and I wasn't sure if that would require tweaking the rollup config as well.

Reply:
> An alternative, and perhaps arguably more sensible, thing to do might be to change the tsc outDir to emit directly to the package root and not do the treemirror copy for the packages that don't need a development build but that seemed like extra work and I wasn't sure if that would require tweaking the rollup config as well.

+1 to that, the minified files in `lit` only save a couple of spaces and newlines.

But also +1 to not trying to do that first to fix the tests. This change seems reasonable to me given that we currently compile to `/development/`, Rollup to the default location, and treemirror in the types, just like in lit-html, etc.

Reply:
I went ahead and removed the `"development"` export conditions from the `lit` package since there's nothing different in those so no need to even include them in `package.json`. I'm still keeping the existing tsc output directory and rollup since I don't want to mess with that, and therefore `"types"` export will still point to `./development/*.d.ts` and the `/development/` directory output will be included in the npm package for those. I think that's a good compromise for now.